README.md

```diff
@@ -1,3 +1,66 @@
 # BibTags
 
-[![Build Status](https://travis-ci.com/tthuem/BibTags.svg?branch=master)](https://travis-ci.com/tthuem/BibTags)
\ No newline at end of file
+[![Build Status](https://travis-ci.com/tthuem/BibTags.svg?branch=master)](https://travis-ci.com/tthuem/BibTags)
+
+## Policies
+- **Maintain consistent order of attributes** of entries. If you want to add a new entry, a good strategy is to copy an existing entry of the same type and adapt it.
+- **Keys are formatted** using author names, publication venue, and year of publication: `<author abbreviation>:<venue><year>`. For example a paper published at `ICSE` in `2005` by `Alice Agda` and `Bob Busy` would have the key `AB:ICSE05`. In the following each key element is explained:
```

Comment:
Maybe we should change the last names of Alice and Bob to start with different letters. Otherwise it is not clear from this example whether the abbreviations abbreviate fornames or surnames.

Reply:
Yes, good idea :)